dual_net.py

```diff
@@ -389,14 +391,18 @@ def export_model(working_dir, model_path):
 
 
 def train(working_dir, *tf_records, steps=None):
+    tf.logging.set_verbosity(tf.logging.INFO)
     if FLAGS.use_tpu:
         tpu_cluster_resolver = tf.contrib.cluster_resolver.TPUClusterResolver(
             FLAGS.tpu_name, zone=None, project=None)
+        tpu_grpc_url = tpu_cluster_resolver.get_master()
 
         config = tpu_config.RunConfig(
-            cluster=tpu_cluster_resolver,
+            master=tpu_grpc_url,
+            evaluation_master=tpu_grpc_url,
             model_dir=working_dir,
-            save_checkpoints_steps=max(600, FLAGS.iterations_per_loop),
+            save_checkpoints_steps=max(200, FLAGS.iterations_per_loop),
+            session_config=tf.ConfigProto(allow_soft_placement=True, log_device_placement=True),
             tpu_config=tpu_config.TPUConfig(
                 iterations_per_loop=FLAGS.iterations_per_loop,
                 num_shards=FLAGS.num_tpu_cores,
```

Comment:
This is appearently ignored by TPUEstimator

https://www.tensorflow.org/api_docs/python/tf/contrib/tpu/TPUConfig

Reply:
Which is?